addons/web/static/src/views/widgets/widget.js

```diff
@@ -66,7 +67,7 @@ Widget.parseWidgetNode = function (node) {
         })
     );
     return {
-        options: evaluateExpr(node.getAttribute("options") || "{}"),
+        options: node.getAttribute("options") || "{}",
```

Comment:
why ?

Reply:
`evaluateExpr` without the record's `evalContext` will throw an error if the expression does not evaluate. And the record is not available in the scope of `parseWidgetNode`.

for example:
```xml
options="{ 'default_journal_id': id }"
```
will fail because id is not defined.

Also, why does it need to be evaluated here? When the template calls `get widgetProps` should be enough. I think the same logic should apply to `parseFieldNode`
discussed with @SimonGenin